trdg/bin/trdg

```diff
@@ -169,9 +170,16 @@ def parse_arguments():
         "--background",
         type=int,
         nargs="?",
-        help="Define what kind of background to use. 0: Gaussian Noise, 1: Plain white, 2: Quasicrystal, 3: Pictures",
+        help="Define what kind of background to use. 0: Gaussian Noise, 1: Plain white, 2: Quasicrystal, 3: random noise vs white, 4: Pictures",
```

Comment:
What is the difference between Gaussian noise and purely random noise?

Reply:
From glancing at the rest of the PR, it does not seem to be used?

Reply:
You are right, i will edit the commnet back.